src/common/meta/src/error.rs

```diff
@@ -112,6 +112,9 @@ pub enum Error {
         source: common_procedure::Error,
     },
 
+    #[snafu(display("Failed to get procedure output"))]
+    ProcedureOutput { location: Location },
```

Comment:
For debugging purpose, better carrying some procedure info. For example, procedure id.